index.js

```diff
@@ -14,14 +21,52 @@ mongoose
     useNewUrlParser: true,
     useUnifiedTopology: true
   })
+
+
   .then(self => {
     console.log(`Connected to the database: "${self.connection.name}"`);
     // Before adding any documents to the database, let's delete all previous entries
     return self.connection.dropDatabase();
   })
+
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    return Recipe.create({
+      title: 'Sushi',
+      level: 'UltraPro Chef',
+      ingredients: ['rice', 'salmon', 'etc'],
+      cuisine: 'japanesse',
+      dishType: 'main_course',
+      image: '',
+      duration: 40,
+      creator: 'Loreto',
+      created: ''
+    })
+      
+    .then(response => console.log('El titulo es:', response.title))
   })
+
+    .then(() => {
+      return Recipe.insertMany(data)
+      .then(recipies => recipies.forEach(elm => console.log('Los titulos son:', elm.title)))
+    })
+
+  
+    .then(() => {
+      return Recipe.findOneAndUpdate({ title: 'Rigatoni alla Genovese' }, {duration: 100}, {new: true})
+        .then(response => console.log('En verdad tardan:', response.duration))
+    })
+
+    .then(() => {
+
+    return Recipe.deleteOne({ title: 'Carrot Cake' })
+        .then(res => console.log('Oh vaya, no nos queda:', res))
+      
+    })
+ 
   .catch(error => {
     console.error('Error connecting to the database', error);
   });
+
+
+  // mongoose.connection.close(console.log('Adeuuus'))
```

Comment:
```suggestion
  // 
  .then(() => mongoose.connection.close(console.log('Adeuuus')))
  .catch(error => {
    console.error('Error connecting to the database', error);
  });
```
La línea del cierre está correcto, solo te faltó simplemente manejarla dentro de una promesa para que la haga al final.